src/App/pages/ProductPage/ProductPage.tsx

```diff
@@ -12,70 +13,55 @@ import RelatedItems from "./components/RelatedItems";
 import styles from "./ProductPage.module.scss";
 import NotFoundPage from "../NotFoundPage";
 
-const RELATED_LIMIT = 3;
-
 const ProductPage = () => {
```

Comment:
нужно не забывать про типизацию
```suggestion
const ProductPage: FC = () => {
```